wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -93,17 +93,21 @@ final class StartUIViewController: UIViewController, SpinnerCapable {
     }
 
     // MARK: - Overloaded methods
+
     override func loadView() {
         view = StartUIView(frame: CGRect.zero)
     }
 
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let title = userSession.selfUser.membership?.team?.name {
+            setupNavigationBarTitle(with: title)
+        } else if let title = userSession.selfUser.name {
+            setupNavigationBarTitle(with: title)
+        }
```

Comment:
maybe extract logic to `private var title: String? { ... }`?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e84c1f504843a40d696dac14c39dec7be4c2e823